components/ui/__tests__/ReviewCard.test.tsx

```
@@ -0,0 +1,385 @@
+      
+      // Rerender with same props
+      rerender(<ReviewCard review={mockReview} onPress={onPress} />);
+      
+      // Component should not re-render (React.memo should prevent it)
+      // This is hard to test directly, but we can verify the component is wrapped
+      expect(ReviewCard.displayName).toBe('ReviewCard');
+    });
+  });
+});
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)